pkg/library/ssh/event.go

```diff
@@ -21,12 +21,15 @@ import (
 	"github.com/devfile/devworkspace-operator/pkg/library/lifecycle"
 )
 
-const commandLine = `SSH_ENV_PATH=$HOME/ssh-environment \
-&& if [ -f /etc/ssh/passphrase ] && command -v ssh-add >/dev/null; \
-then ssh-agent | sed 's/^echo/#echo/' > $SSH_ENV_PATH \
-&& chmod 600 $SSH_ENV_PATH && source $SSH_ENV_PATH \
-&& ssh-add /etc/ssh/dwo_ssh_key < /etc/ssh/passphrase \
-&& if [ -f $HOME/.bashrc ] && [ -w $HOME/.bashrc ]; then echo "source ${SSH_ENV_PATH}" >> $HOME/.bashrc; fi; fi`
+const commandLine = `(SSH_ENV_PATH=$HOME/ssh-environment \
```

Comment:
This looks good to me, but in my testing I was able to copy & paste my final [suggestion](https://github.com/devfile/devworkspace-operator/pull/1338#pullrequestreview-2410740468), which did not include a `\` for every line as well as `;` and had slightly different formatting that seemed a bit clearer to me:
```golang
const commandLine = `(
SSH_ENV_PATH=$HOME/ssh-environment && \
if [ -f /etc/ssh/passphrase ] && [ -w $HOME ] && command -v ssh-add >/dev/null; then
    ssh-agent | sed 's/^echo/#echo/' > $SSH_ENV_PATH \
    && chmod 600 $SSH_ENV_PATH \
    && source $SSH_ENV_PATH \
    && if timeout 3 ssh-add /etc/ssh/dwo_ssh_key < /etc/ssh/passphrase && [ -f $HOME/.bashrc ] && [ -w $HOME/.bashrc ]; then
        echo "source ${SSH_ENV_PATH}" >> $HOME/.bashrc
    fi
fi
) || true`
```

WDYT? Do we really need the `\` line breaks and the `;`?

Reply:
Thanks for the suggestion, applied.

Reply:
BTW I added a new check for the `ssh-agent` availability.

Reply:
Awesome, thank you so much @vinokurig & good call on the check for the `ssh-agent` command :+1: